File: services/dispute/src/main/java/com/example/iter/dispute/domain/repository/DisputeResponseRepository.java
package com.example.iter.dispute.domain.repository;

import com.example.iter.dispute.domain.entity.DisputeResponse;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface DisputeResponseRepository extends JpaRepository<DisputeResponse, Long> {
    List<DisputeResponse> findByDisputeIdOrderByCreatedAtAsc(Long disputeId);
}

File: services/dispute/src/main/kotlin/com/example/iter/dispute/domain/repository/DisputeResponseRepository.kt
package com.example.iter.dispute.domain.repository

import com.example.iter.dispute.domain.entity.DisputeResponse
import org.springframework.data.jpa.repository.JpaRepository

interface DisputeResponseRepository : JpaRepository<DisputeResponse, Long> {

    fun findByDisputeIdOrderByCreatedAtAsc(disputeId: Long): List<DisputeResponse>
}
